core/commands/files.go

```diff
@@ -462,7 +462,24 @@ Examples:
 			return
 		case *mfs.File:
 			_, name := gopath.Split(path)
-			out := &filesLsOutput{[]mfs.NodeListing{mfs.NodeListing{Name: name, Type: 1}}}
+			size, err := fsn.Size()
+			if err != nil {
+				res.SetError(err, cmdkit.ErrNormal)
+				return
+			}
+
+			nd, err := fsn.GetNode()
+			if err != nil {
+				res.SetError(err, cmdkit.ErrNormal)
+				return
+			}
+
+			out := &filesLsOutput{[]mfs.NodeListing{mfs.NodeListing{
+				Name: name,
+				Type: 1,
```

Comment:
Should probably use `unixfs.TFile` to avoid magic numbers.

Reply:
Yes, I should directly call the `Type()` function, `1` isn't even the actual value of `TFile`.., thanks!